WordPress/Classes/ViewRelated/System/Action Sheet/ActionSheetViewController.swift

```diff
@@ -45,11 +45,13 @@ class ActionSheetViewController: UIViewController {
         }
     }
 
+    let headerView: UIView?
     let buttons: [ActionSheetButton]
-    let headerTitle: String
+    let buttonHeaderTitle: String
```

Comment:
I'm a bit confused about this name change. It looks like it's now a button title, but it's still the same header title. What it meant to clarify its use? I'm not sure it did. Maybe we can change it back?

Reply:
I can change it back. I just thought that it'd be confusing to have both the header view and something called a header title. Adding `button` was just to try and clarify it is the title for the buttons.